src/ui/settings/defaults.js

```diff
@@ -129,6 +129,10 @@ export default function defaultSettingsProvider() {
       type: 'json',
       description: 'Default <a href="http://leafletjs.com/reference.html#tilelayer-wms" target="_blank">properties</a> for the WMS map server support in the tile map'
     },
+    'visualization:vectormap:showWarnings': {
+      value: true,
+      description: 'Should the vector map show a warning when terms cannot be joined to a shape on the map.'
+    },
```

Comment:
not 100% sure of this. 

Since we're mashing up 2 datasets, the join might not succeed (partially or completely). The odds of having one or more results not being present on the map can be high, especially when the data is not very clean. That's why I introduced an advanced setting to turn these warnings on/off.